Liadi/events-manager
# #152799743 "slated events" template page

#### What this does?
 - Create a page to show a centers, it's details and slated events
 - Remove heavy images 

#### Task to be completed
 - Navigation bar
 - Colors
 - Advanced Search button

#### How to test features?
  - Navigate to https://liadi.github.io/events-manager/template/slated-events.html
 - Test responsiveness using developer tools
 OR
 - Clone repo(for commit history)
 - Ensure branch is 'ch-slated-events-page-template-152799743' 
 - Open browser and 
 - Navigate to [repo base path]/template/slated-events.html
 
#### Background context
 - None

#### Related PivotalTracker stories (id)
 - #152799743

#### Screenshots
###### Toggle to show events
![slated1](https://user-images.githubusercontent.com/29649308/32882201-e609dc58-cab3-11e7-996d-8144744d80c6.PNG)

###### Toggle to hide events
![slated2](https://user-images.githubusercontent.com/29649308/32882202-e6e1ea08-cab3-11e7-9567-b733f9fe27cf.PNG)



###### Responsiveness
![slated3](https://user-images.githubusercontent.com/29649308/32882203-e73dc288-cab3-11e7-8d06-c33aa04f718f.PNG)
![slated4](https://user-images.githubusercontent.com/29649308/32882205-e793dd80-cab3-11e7-96bb-1e129569f050.PNG)

#### Questions
 - None
